.claude/agents/pr-comment-triager.md

```
@@ -0,0 +1,115 @@
+
+For each thread classified STUPID, resolve via GraphQL mutation:
+
+```bash
+gh api graphql -f query='mutation { resolveReviewThread(input: {threadId: "..."}) { thread { isResolved } } }'
+```
+
+Then post a single PR-level summary comment via `gh pr comment N --body "..."` that lists every stupid thread with a one-line reason. Don't post per-thread reply comments — the on-thread resolution is enough, and a single summary reads better.
+
+The summary comment shape:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)